lstchain/mc/sensitivity.py

```diff
@@ -71,40 +72,45 @@ def process_mc(dl1_file, dl2_file, mc_type):
     sim_par = read_sim_par(dl1_file)
     events = pd.read_hdf(dl2_file)
 
-    #Filters:
-
+    # Filters:
+    # TO DO: These cuts must be given in a configuration file
+    # By now: only cut in leakage and intensity
+    # we use all telescopes (number of events needs to be multiplied 
+    # by the number of LSTs in the simulation)
     filter_good_events =  (events.leakage < 0.2) & \
-                          (events.intensity > np.log10(200)) & \
-                          (events.wl > 0.1) & \
-                          (events.tel_id==1)
+                          (events.intensity > np.log10(50))
```

Comment:
"intensity" is the value in linear scale, so just "intensity>50" ?

Reply:
At least in the old simulation files, this was in log-scale, did it change in the meanitme?

Reply:
Now in the dl1 files we have log_intensity and intensity, so yes, must be just "intensity> ..."

Reply:
ok thanks, added!